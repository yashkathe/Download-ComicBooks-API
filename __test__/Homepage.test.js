const { getLatestComics } = require('../index');

test("Home Page", async () => {
    expect(await getLatestComics(1)).toMatchSnapshot();
});