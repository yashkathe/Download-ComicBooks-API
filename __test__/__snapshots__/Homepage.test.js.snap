// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page 1`] = `
[
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/A.X.E.-Judgment-Day-6-2022.jpg?fit=400%2C608&ssl=1",
    "description": "The apocalyptic finale of the event of the year. It’s not that nothing will be the same again—it’s that unless heroes can find a new way to be heroes, everything will be nothing forever.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/oXBsMxUyShs0KFB4uV1GnBUU+qX4tAeV5mTsDGxaIrALQ04N/+WpDdY5SAFIw8al9E2ZjaUgbJ3zQYGfpXzOQLd5eXRsTAf7JP3ybrQ1RC7A32/sTuaBzjPPZWV8MnJdy7x58oq3tUGGWFSXf37vH1As9F/4i8kmPUmW2o/lrHiqODDdVJOuIu27YlA1Rn2e:B9Oi2ep4rd51JJPf9CUINg==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/p52v17kr",
      },
      {
        "MEGA": "https://getcomics.info/links.php/EI6ChJ7IGbKv54vGF0p/evrP1zhYUbHyJCRSWd7fudRL+qdDibDmpwCZCCAq93WDqjNYfXHyYpvDnaEZySu/JaiwljY0pgME1aHyFN1OXNI=:5s7kPTvuZn5FvTKYmgK7Jg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/qvRbWdaPeVhbWRV1MCIHvQUq2EPG/FwuKG6DQGKbyjFkgMdN2gLTCnB7qg7D2WCKaVDJf2rrZ6wzAuQcMcHorVdXJgbr3M6T1v8S3dinXD1X9r7Osxxs0xtd+DoxjmHQbdnXCK5PcH7iqo7BPGQaPCmxn8E6YLjN0Uj4m9QFxpfDOugQ6XTMU2fIBa6+94eq:r8iBTrJzblszTZkmi4WB0g==",
      },
      {
        "ZIPPYSHARE": "https://www105.zippyshare.com/v/9S6kU2Sf/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/axe-judgment-day-2022/6",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "64 MB",
      "Year": "2022",
    },
    "title": "A.X.E. – Judgment Day #6 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Superman-10-Cent-Adventure-1-2003.jpg?fit=400%2C624&ssl=1",
    "description": "It’s the dawn of a new era for the Man of Steel and a great jumping-on point for new readers – and it’s only 10 cents! With the debut of a new Superman creative team, the powerful new Superman villain Amok and the enigmatic Futuresmiths, this is an issue not to be missed – and at this price, you won’t have to! And wait ’til you see the surprise on page 22!",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/XnJl02KMrU4Hjpj+jefVOG68Jbetny15nO/5LhoGQAGbdRtVSEByKP3v0J4tABz5G6HumLzac3KaW5jTv4RRqQIxw6vnsJ4HG5A61En93+OsuC+QK8p+QOycFFZSMOXB0vHzjhCDKHCZn6lAVXCs8Q==:PtehPVU4QiSxHmTYdTVOoA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/xn5fs3r5",
      },
      {
        "MEGA": "https://getcomics.info/links.php/ekekaJTwqnC2OTf+/KlUS1HFXkToa+hTjwuztihhu4Un+/fbRXujXrIMZ+s23BrEvOy5gHlIDxCL74j4G/oZmNa/3BufyHVu4dI+odgdJLQ=:UBGyVb9AbW+deGlsgOFvJg==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/+jNrH8NVOb6mb3kMT0nyR5L4LJIfEpHuLmo60GBXRfdD2I7VhflAqdqoadl4OAKrKI8A6JR05Ycfg5UWO9EjS7geIn2pLMh6pyySJS7fQ/SkO8Ge62pZGBvt4/MIVYASSj5dV4O0egEJrmRmKEeW1Q==:O0NsM/f+w+jpFt7eRKrFpQ==",
      },
      {
        "ZIPPYSHARE": "https://www14.zippyshare.com/v/lAakpxwu/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "7 MB",
      "Year": "2003",
    },
    "title": "Superman 10-Cent Adventure #1 (2003)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Evil-Dead-1-4-2008.jpg?fit=400%2C618&ssl=1",
    "description": "Verheiden and Bolton present an exciting expansion on the classic horror film that introduced us to the powerful Book of the Dead, the relentlessly violent deadites, and Ash — one resilient, blood-stained survivor. Now an iconic horror hero, relive Ash’s first visit to the cabin that brought him face to face with the delectably deranged deadites who possessed his girlfriend and friends . . . and turned “the perfect place to get laid” into a house of fear and fury. Return to the original nonstop gore-fest and experience the thrills, gags, and gagging anew, with unexpected new directions, additional scenes, and Bolton’s jaw-dropping new paintings.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php//oZtMmFvUlBkPqtPDFUfWERnKHnChiG8jnI1mkurmOacq7OVAm9gJHNmyy+LcOR+FQ4YoYjVX/xf3TGKL9AJPUIZnp+AtsBsj8mq/eJZ9HbxGPcsfLKHbQs7zDhdzJFR:fXlPdjgScIYLlh4KqZxp+w==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/gj8hxoj1",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/xDG5SJqt4z0lGUedYuRrUO1r4Vb4agM44ipmkMMkxfaclu46bBCA/uCL0wc6CTfr+bqFiMz9o9QjeYHWVIJkbqv0qH/LedgwPKj/VBE/kbVDinYQkUmlb52FHBe6ZnmT:RA0eBzuTEudg7nqtX3XN4g==",
      },
      {
        "ZIPPYSHARE": "https://www14.zippyshare.com/v/EurkrL2P/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "47 MB",
      "Year": "2008",
    },
    "title": "Evil Dead #1 – 4 (2008)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/08/Deadline-1-36-1988-1991.jpg?fit=400%2C578&ssl=1",
    "description": "Deadline is a comics magazine published from 1988-1995. It is famous for being the debut of Tank Girl as well as reprinting American independent comics.",
    "downloadLinks": [],
    "info": undefined,
    "title": "Deadline #1 – 50 (1988-1993)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/The-Vertical-Sea-2022.jpg?fit=400%2C571&ssl=1",
    "description": "With a good job as an elementary school teacher and a love for her partner, India’s life seems okay at face value. However, with a chronic mental illness that causes her to have panic attacks regularly, each day can be a struggle. With the threat of having her class taken from her, the pressure is building, and India needs to face her problems head on and take action.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/HVLV36zF6PHtCeoscKT5flb8FKTjYPw3xRxQD8AQN1j0BLCL6QcokkQsbvPRVvDltoJt+H2ZV1LhPlh0+hsTeyEzyIhV1yW7y1xJm/d1N1qbjlhYSN6AK1JTjBK5hClF2Xa16nbiLX30AdDappsno0h/k6jnU+NucF3rek/zsM+NhUyiNGilS3TpvUZURzKw:R5DHgp6O7OBADdRKPtBVVQ==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/07lk8fa5",
      },
      {
        "MEGA": "https://getcomics.info/links.php/vqivLDcF8o3oIjCJD8RZb+sYXknLG3XXAuGteHYJio2h0Yu/AoWdQqn6yI6Ya5o6eZXokdprqoun0rbMbgrMBdzLfcmLvqVRVwNnO0J/xfI=:cK7bFC3Ogb+r7SiV0L9r8g==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/IlGeyPOTFJbhq6eo/FCTTMNLp5wmES1z9LbhywioZVe79aGMVl1OgB0a3vcslgUYSI6hkMWxuPO0P//4aHm3qQ0J7eumPd7KkrgfdSLhMNOl4xRv9SStkaF8iIKUcsdfrGer99bIeKjvdOQB0obkpYRiEhjzE5hYXYQoGC6yZGVlAvHruhrOOXyiqWESDIJV:3ZYE8bdtJyEvJAhonqYUXA==",
      },
      {
        "ZIPPYSHARE": "https://www28.zippyshare.com/v/ESH8oBwe/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/the-vertical-sea-2022/GN",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "242 MB",
      "Year": "2022",
    },
    "title": "The Vertical Sea (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/The-EC-Archives-Crime-Patrol-Vol.-1-2022.jpg?fit=400%2C534&ssl=1",
    "description": "Presenting tales of horrific crimes, grisly murders, and bizarre homicide cases featuring the titanic artistic talents of Johnny Craig, Sheldon Moldoff, H. C. Keifer, Ed Walden, Ann Brewster, and Stan Ashe. Collects Crime Patrol issues #7–#11.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/i2hRhquKWkQYct7bw5mF3x0lDlNJejzvZnfB/xswXyuUjsEeA8XSieK2Q+1AHvR4iCdYT4Dyf9cTF29KU4D27rmcqtoFRIuyw/Q56WJJMKiMCLTcehuTqKJMj82y8qo3OmjZHx60dc18a96tZ7u8jNjy8KR6tbMF6pF6u4QL8iUJOqheFrTSmZZBkAlUBLhg:5FiJ1A3DaxSbsIXsgUvvAA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/rrc18co3",
      },
      {
        "MEGA": "https://getcomics.info/links.php/9vqWr0rvv14aNqz6DJGZ5H8gQ69oE9hotY/xhXjYBoJ6ZpiU8BGrBXlyFjr3YRUZn5ax/I1vy1iGEr/Y1qr7J/6wT8pp2M5PoeIZ/w+SKTA=:HAJMcGQUTHk1dSzlm0/oHw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/vdcrrkmHe6A2pJKYc22L6XEY0exfqM4HhP2iAyNsuDgjnDagQFoD8suJa4TPw1rhjps8W31KMPPlPW/C+TP94j0bp3KyFPKUABMKlEbsJIt7ne5ooWmkE+bGPnEpAZmiJbzQ9VndGm1Qtn9Xc7Hrle39ZxveF7ASVhKAITUUrM2+SZK3RIkMEgaU+SJJG5lQ:QxXs3DU8YCFETpOR0mu/wQ==",
      },
      {
        "ZIPPYSHARE": "https://www28.zippyshare.com/v/BZzMGoS5/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "379 MB",
      "Year": "2022",
    },
    "title": "The EC Archives – Crime Patrol Vol. 1 (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Steeple-Vol.-3-Thats-the-Spirit-2022.jpg?fit=400%2C615&ssl=1",
    "description": "A new year begins in Tredregyn, and a Satanic ritual goes awry – bringing the filming of a cozy TV detective drama to town. Unfortunately, this particular Pandora’s box contains a few things a lot less pleasant than casual jobs and showbiz glitz. Plus, in a special backup feature, a teen sleuth puts the cat among the parish pigeons as she attempts to solve the disappearance of a glamorous author.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/bjzmih6KsRYMISlAKghoPZUdiOXRieV80cyxHqqpSBMDsLPXsTG0rjj5jreQpoUlZfvpZK0Z4NGpXKVhoug1uGWA3ORpKTZuU6h02kd4Ea1FkjOc16t6hJI7gBfHce9+Yfl1bFaEYlgBrTAgBEhetmUffgQG8I3/Zv8KintHV0j4612WdQKT2gCbCk7m2/flhslgDW3MGHAjf6Dj1AZNLQ==:AcJlDxjsQFcsXWElKJH0GA==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/68ucrv5s",
      },
      {
        "MEGA": "https://getcomics.info/links.php/zt3qQz/ZP1oxOO2rTDG6ECMyRe4dMw95EV3PEz3VDe7mZxIcvAZyLkj5keatiEsUSj/SHFBtci69u3Bv9+y/TOyi2ailNBr+/ZqQPiOUEVY=:mTWlNwru8avO+KRovlPrAQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/I0lie4XPzDannVDQNrxoNfcNN4JxCduh2vhf7GxZlL8cDcNKont/IP0Ppw/JpDUjZh7ugQz159DtL2KSzlvBXj2E7teYB4zae2OAE/8Pw3OaMsONY9ycksmAKsA89AjtJcvFxOJx3hsuZ//U5t8gN2/vvYfDXZ243j+1Uz+YE0vSWzvhft1WBnxMMsJrnyHVViuQ7wiFkzme6iEu64fIvA==:aVaQht8aJ+BHCjdI9qNRyA==",
      },
      {
        "ZIPPYSHARE": "https://www28.zippyshare.com/v/baucqBhv/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/steeple-vol-3-thats-the-spirit-2022/GN",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "297 MB",
      "Year": "2022",
    },
    "title": "Steeple Vol. 3 – That’s the Spirit (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Groo-Meets-Tarzan-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "With cartoonist Sergio lost and running from hungry lions in Chula Vista’s Jungle Safari Land and script writer Mark doing panels all day at Comic-Con International, how will this comic get finished? Legendary Tarzan artist Thomas Yeates swings in to help! Plus—the legendary Rufferto backup strips from the comic series and an introduction by Mark Evanier are also included! The confident lord of the jungle meets the dangerous master of cheese dip! Collects Groo Meets Tarzan comics #1 to #4.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/UmlX65n+FZIT65THsOJAd8DuWifD7P8RFnA98jNSJoIWr29allji9jkQijzuRZB7JZkQRIvT6UTG8rOSDiYINGtGkwq1ofjy7PBtH7oAAPwP2lEBMHDVP+IRNacOFYxf9uYzrRzghsQiqRYD7B16MwnMaXllrFRnS/9JxI8q9mq/NvJj47O5jBO6q0X2H8BidN4S+aQuyOwRpzQ5osDNI/XEW0f0HK3tmv+hIVjAo8U=:1czKmEqDDMX7MzQwnV3ZYg==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/05b7nioo",
      },
      {
        "MEGA": "https://getcomics.info/links.php/TN+uppaSwZoeLPEZ4JQM0DlDudFp9Tub/ksVARnGzs1RFirEY+KXnx0gTyenV0RfidVxBwjAexOYexErEIr3q2fU/kzKZBXhh1XCRDNZgWI=:i8fwILjfOrltmGtIAU8qZQ==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/cYNSk27eDtcEUs4tNXUmcwByExBO33fz7WXmO198+ejtZ456dVem409fIi7ijLF00QtbQ8Ek41T1Hv6h8gHsT5NdNqu/5ebxorTagWUCtVaucYb2fFnNpebwvRPH/DACYSNpmEYMK8CtcPekKbxRh/mU/bGJeyFg7fAlYSP0EbZZQ/kCGu6nBpdhmV2n3CIqZEWXIgUMi/Dd+PkgH50ksTPSSeym/tnjxK99N/TeGTY=:yOVbaiosihyHLtHC7dCu4g==",
      },
      {
        "ZIPPYSHARE": "https://www28.zippyshare.com/v/OX6ir0QO/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/groo-meets-tarzan-2021",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "296 MB",
      "Year": "2022",
    },
    "title": "Groo Meets Tarzan (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Hellboy-The-Bones-of-Giants-TPB-2022.jpg?fit=400%2C615&ssl=1",
    "description": "Based on the illustrated novel by Mike Mignola and Christopher Golden, this hardcover collection brings readers into Hellboy’s fight against the Frost Giants with stunning art by Matt Smith (Hellboy and the B.P.R.D: Long Night at Goloski Station, Barbarian Lord, Folklords) and colors by Chris O’Halloran (Folklords, Black Panther, Ice Cream Man). Collects Hellboy: The Bones of Giants issues #1-#4.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/Jeuym6znE8wNJvgVkhBi2be2P5xeZjgoI9bWvcvkyQnTkRABI2bpIgAI6svVx99jkXbzcYhMLxZuIaqOrM+/j3ZiZgRbWvnUbzHzr/AG39yWNybW/DMToldQLZ3yYM1eswX4MKgKFzj59RoX8zA3328TDtpFrGZvKzLE6C4ONBofOYIxDChQfRLSF6tGNRqY/OUxXL0WgdnMffkKVNNtvg==:BospxuK64BwK83KiPNq0Sg==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/421qr8n5",
      },
      {
        "MEGA": "https://getcomics.info/links.php/J6ixVXy/WdvpxKYlUxW2KQOOyX2ZTNYt0/0mG0UGm6ZIS/ZL2BrGwLzJYwZam4sNoZtk0AB933od4IUq+ap999/xSApPBIzT3EstLcpmrkE=:mIQOUVn4+kop7dYi5T8vWw==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/2zgqCsz9mm1voOVfM/TcYOPB7en3FJ+YVUMNAF2uPz+frgNyQV3b+aWaCbYrkVtA3qkCAEbpl0BspDAczDct413BvsX6YNX5LC2dWn0HYAWEulNbzYnaddF97WgnG7wGxJlSKKfjnn+yw96bNSG2mE839gQ0tkji97jlDlYaDTsIGvCGvwfMIjtQtLIkWmza81/h5/Z76mXC0ItAJw/zaA==:5peQxLy2jK4YNv/7u7RxmA==",
      },
      {
        "ZIPPYSHARE": "https://www28.zippyshare.com/v/XSe3ej4E/file.html",
      },
      {
        "Read Online": "https://readcomicsonline.ru/comic/hellboy-the-bones-of-giants-2021",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "181 MB",
      "Year": "2022",
    },
    "title": "Hellboy – The Bones of Giants (TPB) (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Halo-Encyclopedia-2022.jpg?fit=400%2C533&ssl=1",
    "description": "The universe of Halo is remarkably vast in scale and astonishingly elaborate in detail, telling rich stories filled with bold characters, breathtaking worlds, and thrilling conflicts. In celebration of the 20th anniversary of Halo, Dark Horse and 343 Industries have teamed up to deliver the most definitive guide to the universe thus far. Spanning over a hundred millennia and encompassing over a hundred thousand light years, the Halo Encyclopedia captures two decades of storytelling with stunning, never-before-seen art and the most detailed exploration of the universe’s many characters, worlds, and technologies to have ever been created",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/RwQFxkBZzlXmmGgOXeLHokxJecZHCCPh0GdiOch0RlfKswQu8dh/p9z0ygc4MsqLHsfpF07HgTqZal9drd8qoRtTLEwqpCtauqt9eRBsnTyu3Xs7Y6SZf6lsqt3dDoKOyDiDmbQQl3EvCyJe+ERFiP/PBPr4xtrW/NYAhx16RRErC3AcKEdHzO5MsTP05QL6:pU07OvMy24ETdaESsEWt5g==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/eysbkwra",
      },
      {
        "MEGA": "https://getcomics.info/links.php/eNN2DCfL1RnmeebG5mR4+tsCZdAwBxDr4mOs84zxLAdHfYKnILDEn7GDu1ZWPj26vZ3ykMQdXde179+de204WwTrDB5rsZGmH5zKf7w4nW0=:XPsCS0n3aSoD8VQnLb1dsA==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/a6XsTL/AD+nY8s9mrTdKdSJtX1R3lCvIyq5moqynqg4uBN5l3+BmaYVwvT+X49s/G4i+nh58Qj1VLwyq+zak78EIY+PJcc3xSrYDaacyzZMMqa504gX4KyAFZPYs3HIweqDI24szr1gotqD/vySBxMeHmk5SjYqElhtS1AODfr8=:ZuZzamFvmILLA0kVK/YP1Q==",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "549 MB",
      "Year": "2022",
    },
    "title": "Halo Encyclopedia (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Ghost-Rider-Epic-Collection-Vol.-1-Hell-on-Wheels-2022.jpg?fit=400%2C615&ssl=1",
    "description": "In 1972, one of the most iconic characters in comic book history, the flame-skulled Ghost Rider, burned his demonic presence into readers’ minds. Forevermore, a legion of fans were addicted to the Rider’s combination of hell-on-wheels drama and action-horror adventure. And it begins here when Johnny Blaze makes a deal with the devil to save his friend’s life. The payment due? Transformation into the Ghost Rider. The stories that follow will take the horror hero to Hell to battle Satan, pit him against the tempting Witch-Woman and team him with Daimon Hellstrom, Son of Satan. Written by Gary Friedrich and Tony Isabella and gloriously illustrated by Mike Ploog, Tom Sutton and Jim Mooney. COLLECTING: Marvel Spotlight (1971) 5-12, Ghost Rider (1973) 1-11, Marvel Team-Up (1972) 15",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/5POUcYVJ3b4tr45V7uFcW/TP30zjq1hVdr8cIL2SX2mjvJOQe96zzxJiIeUl726BN51sJwkZ/6yg/1mkHUFViM22TgOdShD0PXzdX47OjzbcPgnLvUEywWafZrVt8ptvBg+8FgFrq/0zE1i8b1LokStQBG8XQxdFfZTzcw02gt4FasW2CALN569OZGdYSFmSeehvzvm/mSd779GBC9pDIA==:mcywEZ+GdqdW3OhWS0JR7g==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/ila57kot",
      },
      {
        "MEGA": "https://getcomics.info/links.php/LA8EKdaWD8VWzOHPf8QSamcQusz9j2An7s7CmSvhaH27QcXcJZAvBRy6t+idxKsOjoEp45AZOObnwQb8XpcpFVCdD6dqs5y4Ui2QISIDNh4=:CYt5mzJYbSk7fSigFc5ZVA==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/yngP0anf3d8XnkS5zWLBApESL/rBfG0gc5NBIZ6ODp+n+bGO1hJcqHehVwYXx0IL2JsQxgaGErYivUCwzleG57jK2V44D3ymN2DxKNJ6PPzpd6D3HBTZvEx401NFwpqG8sASgixo6dBZPrp5HcyEe18nyvgAIpBpbZzuuVXB70QrBfZl7813PQ8SXoczS/wM:xICmspz5BLzmOnjRx4RDfQ==",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "697 MB",
      "Year": "2022",
    },
    "title": "Ghost Rider Epic Collection Vol. 1 – Hell on Wheels (2022)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2022/10/Kabuki-Omnibus-Vol.-1-2019.jpg?fit=400%2C615&ssl=1",
    "description": "The origin, the foundation of the story . . . The very beginning of the acclaimed series created by David Mack. This edition collects the first two original Kabuki volumes: Circle of Blood and Dreams in an easy to read trade paper back . . . the perfect book for fans of Mack and Kabuki, and brand-new Kabuki readers! A young woman code name, “Kabuki” struggles with her identity in near-future Japan. Working as an assassin for a clandestine government body known as “The Noh,” Kabuki executes dangerous individuals before they become national-level threats, but when her biological father begins to compromise the agency she works for Kabuki sets out to eliminate him and starts down a difficult path to her own self-discovery.",
    "downloadLinks": [],
    "info": {
      "ImageFormat": "JPG",
      "Size": "2.5 GB",
      "Year": "2019-2021",
    },
    "title": "Kabuki Omnibus Vol. 1 – 4 (2019-2021)",
  },
  {
    "coverPage": "https://i0.wp.com/getcomics.info/share/uploads/2021/06/Ultimate-Comics-Doomsday-TPB-2011.jpg?fit=400%2C615&ssl=1",
    "description": "Who is the face of true evil in the Ultimate Universe? What is the biggest secret the world doesn’t know? Join the gathered heroes as they embark on an adventure unlike anything seen in Marvel Comics — and that’s a promise! Collects Ultimate Comics Enemy #1-4, Ultimate Comics Mystery #1-4, Ultimate Comics Doom #1-4.",
    "downloadLinks": [
      {
        "Download Now": "https://getcomics.info/links.php/UX7srJOrAEo8Ppg97TK0ovvsB44leVslSkEGaDQOzO0zObvAYLp86yBVQTq29Ty2elTr/wcxaz+98vYhmNpVELFTee4QtAFzSRtiTSip4sPBJuBSn/MtboTctdPUd8aiqASthMVtDDDnt5Vgwz4B9Nq0rUdPpxfF0ujD/+k23BNVNXtqIrVZCmlZHBbS3tSa:hulJt6hlozP8IaMkTq9iIw==",
      },
      {
        "UFILE": "https://getcomics.ufile.io/k4d3ci4w",
      },
      {
        "MEGA": "https://getcomics.info/links.php/4uu2LldDcL0yfcI4e26VDClp2SY9uzsVQEnN1y34TjmzjwZYTFSRINRwuubvu1gGVKFGj4TmyHhYqtu4T5Pe481iADllGDPM8QEHoYpQxC4=:Dc1CKMgvTFhgYTrJvOwO2A==",
      },
      {
        "MEDIAFIRE": "https://getcomics.info/links.php/S2qTIFTxaPbipmJ7N8b8M+ZSlOdk0AAZT7XJ5l30zO2D+kZ56hZM9yMjWYieKVjVKia1S073npJGVTzg8P+CUQ/ucyeo5BqD/gVYXh8S48WD3IsmHjKi3XkkqunmCGAiWv/z8M2R6cLT+kJQUgh0KLTmHwOJG+yQfhK0WjJQV/Fgq0VV85qpD6W3Msaxea3T:bQirhdzB0g4xUVHulMD73A==",
      },
      {
        "ZIPPYSHARE": "https://www50.zippyshare.com/v/QNwEqagW/file.html",
      },
    ],
    "info": {
      "ImageFormat": "JPG",
      "Size": "473 MB",
      "Year": "2011",
    },
    "title": "Ultimate Comics Doomsday (TPB) (2011)",
  },
]
`;
